### ISSUE TITLE
hubot not restarted after environment changes

### ISSUE THREAD

**Issue by emanuelis** [brianshumate/ansible-hubot#17]
notify:
    - restart hubot

**Comment by brianshumate** [comment]
Sorry, which environment changes are you proposing this restart for?

**Comment by emanuelis** [comment]
Environment variables

**Comment by brianshumate** [comment]
If it's these:

```yaml
- name: Define Hubot environment
  become: True
  become_user: "{{ hubot_admin }}"
  template:
    src: hubot.env.j2
    dest: "{{ hubot_dir }}/{{ hubot_identity }}.env"
  tags:
    - hubot_environment
```

then the `start-hubot` script does not yet exist so restarting would not be possible.

I think it is restarting in every place appropriate as-is unless you're seeing some kind of problem where it is not?

**Comment by emanuelis** [comment]
Imagine, that after some time of running Hubot you change the password:
```
hubot_env:
  HUBOT_XMPP_HOST: "host"
  HUBOT_XMPP_PORT: "5222"
  HUBOT_XMPP_USERNAME: "hubot@server"
  HUBOT_XMPP_PASSWORD: "new_password"
```
Then the changes are made to environment file but the service is not restarted. It is bad.

Two last lines help:
```
- name: Define Hubot environment
  become: True
  become_user: "{{ hubot_admin }}"
  template:
    src: hubot.env.j2
    dest: "{{ hubot_dir }}/{{ hubot_identity }}.env"
  tags:
    - hubot_environment
  notify:
    - restart hubot
```

**Comment by brianshumate** [comment]
I thought it would be something like that. Very good, I'll go ahead and put that in.

Thanks! :smile:

**Comment by brianshumate** [comment]
Fixed in 8f959219d1eb2668d2764e9424c409fb5b51de01
